src/components/Store.js

```diff
@@ -1,28 +1,29 @@
 import React from "react";
-import moment from "moment";
-import localization from "moment/locale/de";
-moment.locale("de", localization);
 
 export const Store = React.createContext();
 const day = new Date();
 const initialState = {
   date: day,
-  convertedDate: moment(day).format("L"),
-  entry: {}
+  entry: {},
+  entriesByDate: []
 };
 
 function reducer(state, action) {
   switch (action.type) {
     case "SET_DATE":
       return {
-        date: action.payload.date,
-        convertedDate: action.payload.convertedDate
+        date: action.payload.date
```

Comment:
SET_DATE should also return all entries, otherwise they won't be in the global state anymore